charts/connect/Makefile

```diff
@@ -0,0 +1,194 @@
+# 1Password Connect Helm Chart - Testing
+
+# Configuration
+CHART_NAME := connect
+NAMESPACE := default
+KIND_CLUSTER_NAME := onepassword-connect-test
+HELM_TIMEOUT := 120s
+
+# Credentials
+OP_CONNECT_CREDENTIALS ?= $(shell echo "$$OP_CONNECT_CREDENTIALS")
+OP_CONNECT_TOKEN ?= $(shell echo "$$OP_CONNECT_TOKEN")
+OP_SERVICE_ACCOUNT_TOKEN ?= $(shell echo "$$OP_SERVICE_ACCOUNT_TOKEN")
+
+# Test fixture values (must be set as environment variables)
+# OP_VAULT_ID - 1Password vault ID for testing
+# OP_ITEM_ID - 1Password item ID for testing  
+# OP_SECRET_VALUE - Expected secret value for testing
+
+# Colors for output
+RED := \033[0;31m
+GREEN := \033[0;32m
+YELLOW := \033[0;33m
+BLUE := \033[0;34m
+NC := \033[0m # No Color
+
+.PHONY: help test-e2e test-e2e-setup test-e2e-run cleanup-test-e2e check-deps revert-ci-files
+
+test-e2e: check-deps test-e2e-setup test-e2e-run cleanup-test-e2e ## Run end-to-end tests (full workflow)
+
+test-e2e-setup: ## Setup test environment
+	@echo "$(BLUE)Setting up test environment...$(NC)"
+
+	# Check if all required environment variables are set
+	@if [ -z "$(OP_CONNECT_CREDENTIALS)" ]; then \
+		echo "$(RED)Error: OP_CONNECT_CREDENTIALS environment variable is required$(NC)"; \
+		exit 1; \
+	fi
+	@if [ -z "$(OP_CONNECT_TOKEN)" ]; then \
+		echo "$(RED)Error: OP_CONNECT_TOKEN environment variable is required$(NC)"; \
+		exit 1; \
+	fi
+
+	@if [ -z "$(OP_SERVICE_ACCOUNT_TOKEN)" ]; then \
+		echo "$(RED)Error: OP_SERVICE_ACCOUNT_TOKEN environment variable is required$(NC)"; \
+		exit 1; \
+	fi
+
+	@if [ -z "$(OP_VAULT_ID)" ]; then \
+		echo "$(RED)Error: OP_VAULT_ID environment variable is required$(NC)"; \
+		echo "$(YELLOW)Set OP_VAULT_ID to your 1Password vault ID for testing$(NC)"; \
+		exit 1; \
+	fi
+	@if [ -z "$(OP_ITEM_ID)" ]; then \
+		echo "$(RED)Error: OP_ITEM_ID environment variable is required$(NC)"; \
+		echo "$(YELLOW)Set OP_ITEM_ID to your 1Password item ID for testing$(NC)"; \
+		exit 1; \
+	fi
+	@if [ -z "$(OP_SECRET_VALUE)" ]; then \
+		echo "$(RED)Error: OP_SECRET_VALUE environment variable is required$(NC)"; \
+		echo "$(YELLOW)Set OP_SECRET_VALUE to the expected secret value for testing$(NC)"; \
+		exit 1; \
+	fi
```

Comment:
`[nit]`

This can be extracted into a target that's not exposed. You can pass variable name, expected string name (e.g. "item ID") and the variable you want to verify to it.